server/superdesk/io/commands/update_ingest.py

```diff
@@ -221,6 +228,8 @@ def process_anpa_category(item, provider):
                     if anpa_category['is_active'] is True \
                             and item_category['qcode'].lower() == anpa_category['qcode'].lower():
```

Comment:
we don't need extra lower() for `anpa_category['qcode']` as it's already lower in case.